.github/workflows/bot-nitrogen.yml

```diff
@@ -1,20 +1,22 @@
 name: "[Bot] Testing with 'Nitrogen'"
-run-name: Test with 🤖 Nitrogen by @${{github.actor}} with ${{inputs.family}}
+run-name: Test with 🤖 Nitrogen by @${{github.actor}} family=${{inputs.families}} currency=${{inputs.currencies}}
 on:
   workflow_dispatch:
     inputs:
-      family:
-        description: coin family to filter with (if any)
+      ref:
+        description: ref branch to run on
+        required: true
+      families:
+        description: coin family filter, separated with commas (,)
+        required: false
+      currencies:
+        description: currency ids to filter, separated with commas (,)
         required: false
       speculos_websocket:
         description: check to use speculos with Websocket
         type: boolean
         required: false
         default: false
-  push:
```

Comment:
why did you remove on: push ?

Reply:
We always use workflow dispatch

Reply:
@valpinkman what is the purpose of this `push` configuration?

At first I thought:

> It looks like this (and similar things on other workflows) are here for a reason and used for easier support and maintenance.

But since these workflows are triggerd manually through workflow dispatch, it doesn't seem needed because we can select the branch from which to execute the workflow on github if we want to test our updates before merging them.

If this is still needed, could you provide details so it can be included in a comment? Thanks

Reply:
I never said it was mandatory, I was actually asking why it was removed and the answer is enough. You can remove the on push trigger if it is indeed never used 👍